wiki/Tournaments/RDOC/2019/en.md

```diff
@@ -0,0 +1,155 @@
+---
+tags:
+  - RDOC
+  - RDOC2019
+---
+
+# Royal Danish osu!Cup 2019
+
+![RDOC 2019 logo](img/logo.png)
+
+The **Royal Danish osu!Cup 2019** (***RDOC 2019***) is an ongoing Danish 1v1 double-elimination osu!standard tournament hosted by ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742), ![][flag_DK] [MadX](https://osu.ppy.sh/users/3295048), and ![][flag_DK] [MASQQ](https://osu.ppy.sh/users/5351497). This is the fifth installment of the Royal Danish osu!Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2020-01-10/2020-01-24 |
+| Qualifiers | 2020-02-07/2020-02-16 |
+| Live drawings | *to be determined* |
+| Round of 32 | 2020-02-29/2020-03-01 |
+| Round of 16 | 2020-03-07/2020-03-08 |
+| Quarterfinals | 2020-03-14/2020-03-15 |
+| Semifinals | 2020-03-21/2020-03-22 |
+| Finals | 2020-03-28/2020-03-29 |
+| Grand Finals | 2020-04-04/2020-04-05 |
+
+## Prizes
+
+| Placing | Prize(s) |
```

Comment:
```suggestion
| Placing | Prizes |
```

Reply:
same as all the other tournaments i think this is fine